src/features/loading/LoadingScreen.tsx

```diff
@@ -290,11 +291,12 @@ class LoadingScreen extends Component<TLoadingProps, TLoadingState> {
         // And reply that we're all done.
         ipcRenderer.send('appquitdone')
       }
+
+      closeServeStatic()
```

Comment:
Similar to the comment above, we should call this in `main.ts` on the event when the app shutting down

Reply:
It has been updated